src/NuGet.Core/NuGet.Common/PathUtil/PathUtility.cs

```diff
@@ -6,13 +6,14 @@ namespace NuGet.Common
 {
     public static class PathUtility
     {
+        private static readonly Lazy<bool> _isMacOSCaseSensitive = new Lazy<bool>(IsMacOSCaseSensitive);
```

Comment:
There should be a public property for this that returns the value of the lazy, the same as the other helpers such as IsMono.